packages/ai-google/src/node/google-language-model.ts

```diff
@@ -184,6 +200,10 @@ export class GoogleModel implements LanguageModel {
                             functionDeclarations
                         }]
                     }),
+                    thinkingConfig: {
+                        // https://ai.google.dev/gemini-api/docs/thinking#summaries
+                        includeThoughts: true,
+                    },
```

Comment:
Do we set thinking on "on" on purpose? It's a bit cumbersome to turn it off again
```
"ai-features.modelSettings.requestSettings": [
  {
    "scope": {
      "providerId": "google"
    },
    "requestSettings": {
      "thinkingConfig": {
        "includeThoughts": false
      }
    }
  }
]
```

On the other hand, turning it on would be equally as difficult